subprojects/gradle/build-environment/src/main/kotlin/com/avito/utils/gradle/EnvironmentInfo.kt

```diff
@@ -0,0 +1,84 @@
+package com.avito.android.sentry
+
+import com.avito.git.Git
+import com.avito.kotlin.dsl.lazyProperty
+import com.avito.logger.DefaultLogger
+import com.avito.logger.LogLevel
+import com.avito.logger.Logger
+import com.avito.logger.LoggerFactory
+import com.avito.logger.LoggingDestination
+import com.avito.logger.handler.DefaultLoggingHandler
+import com.avito.utils.gradle.Environment
+import com.avito.utils.gradle.internal.EnvironmentInfoImpl
+import org.gradle.api.Project
+import org.gradle.api.provider.Provider
+
+/**
+ * Use [Project.environmentInfo] to gain instance
+ */
+interface EnvironmentInfo { // TODO: merge with BuildEnvironment and EnvArgs
+    val node: String?
```

Comment:
What is the node property?

Reply:
It's used to describe a current host or machine in CI.

```kotlin
when (environment) {
            is Environment.Local -> gitUserEmail ?: userName()
            is Environment.Mirakle -> userName()
            is Environment.CI -> teamcityAgentName()
            is Environment.Unknown -> "unknown"
        }
```

It's a workaround for metric because we don't have a unique build id like in build scan.